test/crypto.js

```diff
@@ -72,20 +72,53 @@ describe('crypto', () => {
       assert.equal(input, decoded)
     })
   })
-  describe('sign and verify', () => {
-    it('should work together', async () => {
-      let privateKey = Buffer.from(privateKeyAsHex, 'hex')
-      let signature = Crypto.sign(validUnsignedTx, privateKey)
-      let pub = Buffer.from(Crypto.decodeBase58Check(publicKey.split('$')[1]))
-      let verified = Crypto.verify(validUnsignedTx, signature, pub)
-      assert.isTrue(verified)
+
+  describe('sign', () => {
+    it('should produce correct signature', () => {
+      let s = Crypto.sign(txBinary, privateKey)
```

Comment:
Use `const` instead of `let` wherever possible (please disregard that old code does it wrong)